Sources/Build/BuildPlan.swift

```diff
@@ -1628,8 +1638,15 @@ public class BuildPlan {
                     guard let binaryTarget = target.underlyingTarget as? BinaryTarget else {
                         throw InternalError("invalid binary target '\(target.name)'")
                     }
-                    if case .xcframework = binaryTarget.kind, let library = self.xcFrameworkLibrary(for: binaryTarget) {
-                        libraryBinaryPaths.insert(library.binaryPath)
+                    switch binaryTarget.kind {
+                    case .xcframework:
+                        let libraries = try self.parseXCFramework(for: binaryTarget)
+                        libraries.forEach { library in
```

Comment:
Nit: what is our stance on `for _ in` vs `forEach` in the SwiftPM codebase?  Historically I think we've been using `for _ in`, which I personally find much easier to read, but we should probably choose one and use the same in the codebase.